client/src/main/scala/it/cwmp/client/view/room/RoomFXController.scala

```diff
@@ -35,6 +35,14 @@ class RoomFXController(strategy: RoomFXStrategy) extends FXController with FXVie
   private var pr_et_roomID: TextField = _
   @FXML
   private var pub_et_numPlayer: Spinner[Integer] = _
+  @FXML
+  private var pr_cr_btnCreate: Button = _
+  @FXML
+  private var pr_cr_btnReset: Button = _
+  @FXML
+  private var pr_et_btnEnter: Button = _
+  @FXML
+  private var pub_et_btnEnter: Button = _
```

Comment:
Questi nomi sono a dir poco **orribili**
Sarebbe bene cambiarli subito! In qualcosa di intelleggibile in camelCase